test/test_utils.dart

```diff
@@ -1,6 +1,6 @@
-import 'package:treasure_of_the_high_seas/model/card/a_rival_ship.dart';
+import 'package:treasure_of_the_high_seas/model/card/basic/a_rival_ship.dart';
 import 'package:treasure_of_the_high_seas/model/card/card.dart';
-import 'package:treasure_of_the_high_seas/model/card/plunder_a_wreck.dart';
+import 'package:treasure_of_the_high_seas/model/card/basic/plunder_a_wreck.dart';
```

Comment:
When we start working with many cards in one file, we might want a way to avoid importing each card individually, I might make a new issue for this

Reply:
Agree, worth doing under the separate issue.